angularjs-portal-frame/src/main/webapp/frame.html

```diff
@@ -71,5 +71,6 @@
 <script type="text/javascript" src="js/filters.js"></script>
 <script type="text/javascript" src="js/ui-bootstrap.js"></script>
 <script type="text/javascript" src="js/ga.js"></script>
+<script type="text/javascript" src="my-app.js"></script>
```

Comment:
I could ask the same question why the other scripts are in a js directory :)

The angular style guides recommend organizing source by function, rather than by language. 'my-app.js' is seemingly the linchpin, so it's at the root of the web app. App developers would then have sub-directories by app function containing controllers/services/partials/etc.